runtime/calamari/src/xcm_config.rs

```diff
@@ -271,9 +271,8 @@ impl pallet_xcm::Config for Runtime {
     type RuntimeEvent = RuntimeEvent;
     type SendXcmOrigin = EnsureXcmOrigin<RuntimeOrigin, LocalOriginToLocation>;
     type XcmRouter = XcmRouter;
-    type ExecuteXcmOrigin = EnsureXcmOrigin<RuntimeOrigin, LocalOriginToLocation>;
+    type ExecuteXcmOrigin = EnsureXcmOrigin<RuntimeOrigin, ()>;
```

Comment:
Why removed `LocalOriginToLocation`?

Reply:
This is how it was before this commit.
We don't need the execute extrinsic, first because there's no requirement for it like send, but also because it would break our outgoing transfers barrier on xTokens